### PR TITLE
Updates for Chrome 122 beta

### PR DESCRIPTION
**Pull request by Elchi3**
The [Open Web Docs BCD collector](https://github.com/openwebdocs/mdn-bcd-collector) v10.7.1 found new features shipping in Chrome 122 beta which was released yesterday. Currently, the collector covers about 82% of BCD, so the following list might not be exhaustive. Also, if a feature is in Chrome Canary/behind origin trials/enrollment, it is not considered here.

With this PR, BCD considers the following 44 features as shipping in Chrome 122:

- api.IdentityCredential.disconnect_static
- api.Navigator.storageBuckets
- api.StorageBucket
- api.StorageBucket.caches
- api.StorageBucket.estimate
- api.StorageBucket.expires
- api.StorageBucket.getDirectory
- api.StorageBucket.indexedDB
- api.StorageBucket.name
- api.StorageBucket.persist
- api.StorageBucket.persisted
- api.StorageBucket.setExpires
- api.StorageBucketManager
- api.StorageBucketManager.delete
- api.StorageBucketManager.keys
- api.StorageBucketManager.open
- api.URLPattern.hasRegExpGroups
- api.WebGL2RenderingContext.drawingBufferFormat
- api.WebGL2RenderingContext.drawingBufferStorage
- api.WebGLRenderingContext.drawingBufferFormat
- api.WebGLRenderingContext.drawingBufferStorage
- api.WorkerNavigator.storageBuckets
- css.types.color.rgb.mixed_type_parameters (not in this PR but added to BCD manually previously)
- css.types.color.rgba.mixed_type_parameters (ditto)
- javascript.builtins.Iterator.Iterator
- javascript.builtins.Iterator.drop
- javascript.builtins.Iterator.every
- javascript.builtins.Iterator.filter
- javascript.builtins.Iterator.find
- javascript.builtins.Iterator.flatMap
- javascript.builtins.Iterator.forEach
- javascript.builtins.Iterator.from
- javascript.builtins.Iterator.map
- javascript.builtins.Iterator.reduce
- javascript.builtins.Iterator.some
- javascript.builtins.Iterator.take
- javascript.builtins.Iterator.toArray
- javascript.builtins.Set.difference
- javascript.builtins.Set.intersection
- javascript.builtins.Set.isDisjointFrom
- javascript.builtins.Set.isSubsetOf
- javascript.builtins.Set.isSupersetOf
- javascript.builtins.Set.symmetricDifference
- javascript.builtins.Set.union

I think all of this is in line with what https://chromestatus.com says currently.

Again, I hope this auto-generated PR is useful to update BCD for the new Chrome 122 more easily and faster. If you have feedback, let me know! /cc @chrisdavidmills

### CODE REVIEW

File: api/IdentityCredential.json [hunk 1]
```diff
@@ -33,6 +33,40 @@
           "deprecated": false
         }
       },
+      "disconnect_static": {
+        "__compat": {
+          "description": "<code>abort()</code> static method",
```
Comment (Elchi3) [inline api/IdentityCredential.json:38]:
```suggestion
          "description": "<code>disconnect()</code> static method",
```

**Review by chrisdavidmills — COMMENTED** [review]
@Elchi3 I checked the features added in this PR against the features listed as enabled in Chrome 122 beta release (see https://chromestatus.com/roadmap), and this is what I found:

## Found by the collector and listed as enabled in the 122 beta

- [Feature: FedCM: disconnect](https://chromestatus.com/feature/5202286040580096)
  - `api.IdentityCredential.disconnect_static`
- [Feature: Iterator helpers](https://chromestatus.com/feature/5102502917177344)
  - `javascript.builtins.Iterator.Iterator`
  - `javascript.builtins.Iterator.drop`
  - `javascript.builtins.Iterator.every`
  - `javascript.builtins.Iterator.filter`
  - `javascript.builtins.Iterator.find`
  - `javascript.builtins.Iterator.flatMap`
  - `javascript.builtins.Iterator.forEach`
  - `javascript.builtins.Iterator.from`
  - `javascript.builtins.Iterator.map`
  - `javascript.builtins.Iterator.reduce`
  - `javascript.builtins.Iterator.some`
  - `javascript.builtins.Iterator.take`
  - `javascript.builtins.Iterator.toArray`
- [Feature: Set methods](https://chromestatus.com/feature/6280344932450304)
   - `javascript.builtins.Set.difference`
   - `javascript.builtins.Set.intersection`
   - `javascript.builtins.Set.isDisjointFrom`
   - `javascript.builtins.Set.isSubsetOf`
   - `javascript.builtins.Set.isSupersetOf`
   - `javascript.builtins.Set.symmetricDifference`
   - `javascript.builtins.Set.union`
- [Feature: Storage Buckets API](https://chromestatus.com/feature/5739224579964928)
  - `api.Navigator.storageBuckets`
  - `api.StorageBucket`
  - `api.StorageBucket.caches`
  - `api.StorageBucket.estimate`
  - `api.StorageBucket.expires`
  - `api.StorageBucket.getDirectory`
  - `api.StorageBucket.indexedDB`
  - `api.StorageBucket.name`
  - `api.StorageBucket.persist`
  - `api.StorageBucket.persisted`
  - `api.StorageBucket.setExpires`
  - `api.StorageBucketManager`
  - `api.StorageBucketManager.delete`
  - `api.StorageBucketManager.keys`
  - `api.StorageBucketManager.open`
  - `api.WorkerNavigator.storageBuckets`
- [Feature: URLPattern: hasRegExpGroups](https://chromestatus.com/feature/4761338127843328)
  - `api.URLPattern.hasRegExpGroups`
- [Feature: WebGL drawingBufferStorage](https://chromestatus.com/feature/5146687245123584)
  - `api.WebGL2RenderingContext.drawingBufferFormat`
  - `api.WebGL2RenderingContext.drawingBufferStorage`
  - `api.WebGLRenderingContext.drawingBufferFormat`
  - `api.WebGLRenderingContext.drawingBufferStorage`

## Found by the collector, but not listed in the 122 beta list

- `css.types.color.rgb.mixed_type_parameters` (not in this PR but added to BCD manually previously)
- `css.types.color.rgba.mixed_type_parameters` (ditto)

## Not found by the collector

- [Allow setting IDP login status from same-site subresources](https://chromestatus.com/feature/5086015570313216): This doesn't sound like something that would be automatically detected anyway. It would be added as a manual sub-feature on api.NavigatorLogin.setStatus
- [Feature: Async Clipboard API: Read unsanitized HTML](https://chromestatus.com/feature/5203909459312640): New `unsanitized` option for api.Clipboard.read; would need to be added manually as a subfeature
- [Feature: Back/forward cache NotRestoredReason API](https://chromestatus.com/feature/5684908759449600): I'm pretty sure this is already added to BCD (api.PerformanceNavigationTiming.notRestoredReasons) from when I [attempted to document it](https://github.com/mdn/content/pull/27947) a while ago
- [Feature: CSS @container with unsupported Features never match](https://chromestatus.com/feature/5105423604776960): New behavior for `css.at-rules.container`; should be manually added as a subfeature
- [Feature: CSS Pseudo Element ::backdrop inheriting from Originating Element](https://chromestatus.com/feature/4875749691752448): New behavior for `css.selectors.backdrop`; should be manually added as a subfeature
- [Feature: Do not delete File kind objects in dataTransfer.clearData()](https://chromestatus.com/feature/5735154930941952): New behavior for `api.DataTransfer.clearData`; should be manually added as a subfeature
- [Feature: Interoperable mousedown event cancellation in iframe](https://chromestatus.com/feature/5083240891416576): New behavior for `api.Element.mousedown_event`; should be manually added as a subfeature
- [Feature: RTCRtpSender setParameters() extensions for requesting the generation of a key frame](https://chromestatus.com/feature/5161082937409536): New behavior for `api.RTCRtpSender.setParameters`; should be manually added as a subfeature 
- [Feature: X25519Kyber768 key encapsulation for TLS](https://chromestatus.com/feature/5257822742249472): Not really sure if/how we'd record this one. More of an infrastructure security thing than a web dev feature?

**Comment by Elchi3** [comment]
Thanks again for your detailed look, @chrisdavidmills! Sounds like everything this PR is proposing is correct then and there are few things that could be added as manual additions to BCD as follow-ups. If you're happy with this PR, then I think you could approve and merge :)

**Comment by chrisdavidmills** [comment]
> Thanks again for your detailed look, @chrisdavidmills! Sounds like everything this PR is proposing is correct then and there are few things that could be added as manual additions to BCD as follow-ups. If you're happy with this PR, then I think you could approve and merge :)

Yup, agreed. I will do so ;-)

**Pull request merged**